src/backend/utils/view-helpers.js

```diff
@@ -154,10 +154,6 @@ class ViewHelpers {
   assetPath(asset) {
     return this.urlBuilder(['frontend', 'assets', asset])
   }
-
-  isMainColumn(propertyName) {
```

Comment:
doing it this way causes multiple columns to be "main" - if record had id name and email all those 3 fields were treated as main.

there is a isTitle method in property - I used that and it fixed the bug